### PR TITLE
Rename examples test dir to tflearn

### PR DESCRIPTION
**Pull request by terrytangyuan**
- SKFLOW_EXAMPLE_BASE_DIR -> TFLEARN_EXAMPLE_BASE_DIR
- Removed extra echo

### CODE REVIEW

**Comment by terrytangyuan** [comment]
One flaky test had timeout [here](https://ci.tensorflow.org/job/tensorflow-pull-requests-gpu/2197/). @tensorflow-jenkins Test this please

File: tensorflow/examples/learn/examples_test.sh [hunk 1]
```diff
@@ -28,20 +28,18 @@ then
   DIR="$DIR"/"$TEST_WORKSPACE"
 fi
 
-SKFLOW_EXAMPLE_BASE_DIR=$DIR/tensorflow/examples/learn
+TFLEARN_EXAMPLE_BASE_DIR=$DIR/tensorflow/examples/learn
```
Comment (drpngx) [inline tensorflow/examples/learn/examples_test.sh]: I would have gone for TF_LEARN_EXAMPLE_BASE_DIR, though the filenames in the doc are tflearn.

**Review by drpngx — APPROVED** [review]
Assuming you know what the proper word-breaking is.

File: tensorflow/examples/learn/examples_test.sh (hunk 1 above)
Comment (terrytangyuan) [inline tensorflow/examples/learn/examples_test.sh]: Well in my opinion `TF_LEARN_EXAMPLE` sounds like some TensorFlow examples you can learn. But anyways, I don't think users care about this script so let's just go with this at this moment.

Comment (drpngx) [inline tensorflow/examples/learn/examples_test.sh]: SG

**Pull request merged**